packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -648,9 +648,9 @@ curl -v -X POST \
 
 #### Create User with Non-Default User Type
 
-Creates a user with a specified User Type (see <ApiLifecycle access="ea" /> [User Types](/docs/reference/api/user-types)). The type specification may be included with any of the above Create User operations; this example demonstrates creating a user without credentials.
+Creates a user with a specified User Type (see [User Types](/docs/reference/api/user-types)). The type specification may be included with any of the above Create User operations; this example demonstrates creating a user without credentials.
 
-The User Type determines which [Schema](/docs/reference/api/schemas) applies to that user. After a user has been created, the user currently cannot be assigned a different User Type.
+The User Type determines which [Schema](/docs/reference/api/schemas) applies to that user. After a user has been created, the user currently can only be assigned a different User Type by an administrator via a full replace update.
```

Comment:
I would consider removing "currently", and also moving the word "only" to the part that it pertains to.  (I.e., many other things can be done to the user, but it can be assigned a different type only by blah blah blah.)  Finally, consider "using" instead of "via".

... the user can be assigned a different User Type only by an administrator using a full replace update.

Reply:
I agree with all those except I like 'via' :) .  Changed.